src/FunctionApps/SilverTransforms/Transform/transforms.py

```diff
@@ -0,0 +1,67 @@
+import pymongo
+import smartystreets_python_sdk.us_street
+
+from Transform.geo import cached_geocode
+
+
+def transform_name(raw: str) -> str:
+    """trim spaces, capitalize, etc"""
+    raw = [x for x in raw if not x.isnumeric()]
+    raw = "".join(raw)
+    raw = raw.upper()
+    raw = raw.strip()
+    return raw
+
+
+def transform_phone(raw: str) -> str:
+    """Make sure it's 10 digits, remove everything else"""
+    raw = [x for x in raw if x.isnumeric()]
+    raw = "".join(raw)
+    if len(raw) != 10:
+        raw = None
```

Comment:
Hmm can we come up with a more elegant way to handle phone numbers? I don't know if there is one but throwing out all the instances that don't match 10 numbers exactly feels...I don't know, wasteful? Although, if I talk through it, I don't know if there really is an alternative. If it's less than 10 we probably should throw it out (because we can't just make up the missing number), and if it's over 10, how do we choose which 10 to use (could be the first ten maybe)? Idk, I might be rambling, but just flagging in case there are conversation thoughts. If not, happy to resolve this and fade back into the shadows 🐍

Reply:
Definitely happy to think about more sophisticated phone number handling. The challenge is that a US phone number that isn't 10 digits is not unique and therefore not ideal for record linkage. I'd be happy to consider building out functionality to parse different specific formats. This would allow us to identify an extension and choose how we want to handle it.

Reply:
Maybe the best solution is to accept this for now as an easy/strong starting point, and then as we work on refining record linkage, we can just refine phone number parsing under the hood (if it'll be wrapped in a function that'll allow micro changes without affecting larger code). Extension parsing could be a good follow up ticket.